server/controllers/stock/reports/stock/adjustment_receipt.js

```diff
@@ -28,7 +28,23 @@ async function stockAdjustmentReceipt(documentUuid, session, options) {
     WHERE m.flux_id IN (${Stock.flux.FROM_ADJUSTMENT}, ${Stock.flux.TO_ADJUSTMENT}) AND m.document_uuid = ?
   `;
 
-  const rows = await db.exec(sql, [db.bid(documentUuid)]);
+  const sqlGetVoucherReference = `
```

Comment:
Does every receipt need `sqlGetVoucherReference`?  If so, I would make it a function and put it in the `stock/reports/common.js`.

Like this:
```js
function getVoucherReferenceForStockMovement(documentUuid) {
  const sql = `
    SELECT v.uuid, dm.text AS voucher_reference
    FROM voucher AS v
    JOIN voucher_item AS vi ON vi.voucher_uuid = v.uuid
    JOIN document_map AS dm ON dm.uuid = v.uuid
    WHERE vi.document_uuid = ?
    LIMIT 1;
  `;

  return db.exec(sql, [db.bid(documentUuid)]);
}

exports.getVoucherReferenceForStockMovement = getVoucherReferenceForStockMovement;
```

Then every stock module can import it and just use:
```js
const { getVoucherReferenceForStockMovement }  = require('../common');

// later ..

const results = await q.all([
  db.exec(sql, [db.bid(documentUuid)]), 
  getVoucherReferenceForStockMovement(documentUuid),
]);

// etc
```

This would be more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).